content/blog/2022/10/kubewarden_1_3_release.md

```diff
@@ -0,0 +1,135 @@
+---
+title: "Kubewarden 1.3 is out!"
+authors:
+- Flavio Castelli
+date: 2022-10-26
+---
+
+The Kubewarden development team is happy to announce the release of the Kubewarden 1.3 stack.
+
+In addition to the usual amount of small fixes, this release focused on the following
+themes.
+
+## Improve end users confidence
+
+We want our users to feel confident about using Kubewarden, knowing that good
+development and security practices are being followed by the Kubewarden project.
+We think this is particularly relevant to Kubewarden, given our users trust us
+to keep their Kubernetes clusters secure and compliant.
+
+To achieve that, we've joined the [CLOMonitor](https://clomonitor.io/) initiative.
+This is a project from the Clound Native Foundation which performs a series of
+checks to ensure certain project health best practices are met.
+
+We are proud to announce that Kubewarden is currently A-rated, with a score above
+the 90%. The results are completely transparent and can be found by visiting
+the [Kubewarden dashboard](https://clomonitor.io/projects/cncf/kubewarden)
+on the CLOMonitor website.
+
+## Performance improvements
+
+We have reduced the startup time of Policy Server, the component in charge of
+running all our WebAssembly policies.
+The difference is quite significant when multiple policies are enforced by the
+same `PolicyServer` Kubernetes resource.
+
+To achieve that, we had to perform some refactoring of the internal architecture
+of Policy Server. The good news is that this work paves the way for a feature
+we will deliver shortly: being able to put a constraint on the evaluation
```

Comment:
```suggestion
we will deliver shortly: being able to put a *custom* constraint on the evaluation
```
I'm just concerned that people may think policy evaluation didn't have timeouts (even if it's only the webhook ones)

Reply:
Good point, I'll find a way to address that

Reply:
I removed this section as discussed during the daily